hw/ip/otbn/util/shared/operand.py

```diff
@@ -275,7 +278,7 @@ def make(width: Optional[int],
                                  'but the operand claims to have width {}.'
                                  .format(what, scheme_field.bits.width, width))
 
-        return ImmOperandType(width, shift, signed, pc_rel)
+        return ImmOperandType(width, fixed_offset, shift, signed, pc_rel)
 
     def markdown_doc(self) -> Optional[str]:
```

Comment:
For the doc generation there's a question of how we express how the asm values turns into the encoded value. For shifted values this is left for the reader to infer rather then explicitly stated currently. This is the same for fixed_offset but it may not be so obvious, should we add a 'encoded value is offset by X' to the doc string?

Reply:
@imphil has also talked about how it's important to document this and I agree. I've opened #4388 to track it.

Reply:
Ok, I'll leave things be for this PR and we'll get it sorted in separate PR to address all of #4388